android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -1037,15 +1024,10 @@ protected void onResume()
     super.onResume();
     refreshSearchToolbar();
     setFullscreen(isFullscreen());
-    if (Framework.nativeIsInChoosePositionMode())
+    if (Framework.nativeGetChoosePositionMode() != Framework.ChoosePositionMode.NONE)
     {
       UiUtils.show(mPointChooser);
       mMapButtonsViewModel.setButtonsHidden(true);
-      if (mPointChooserMode == PointChooserMode.NONE)
-      {
-        // BUG: https://github.com/organicmaps/organicmaps/issues/3945
```

Comment:
Rotating the device should be the documented test case for everything, including API calls.